### DISCUSSION TITLE
[NEW RELEASE] Introducing MARS5, open-source, insanely prosodic text-to-speech (TTS) model.

### DISCUSSION THREAD

**Discussion by akshhack** [coqui-ai/TTS#3784] [Show and tell]
![mars-banenr](https://github.com/coqui-ai/TTS/assets/21692676/2a8422d2-c67d-463c-bbb6-f96db2ff7cc6)

Hey community members! 👋 

We, at [CAMB.AI](https://www.camb.ai), are super stoked to announce the open source release of **MARS5**, a new speech emulation model that is able to replicate even extremely tough prosody like sports commentary, anime, movies with just a few seconds of audio reference. 

_Check out our release:_ https://github.com/Camb-ai/MARS5-TTS

Watch our demo here:

https://github.com/coqui-ai/TTS/assets/21692676/c70429dc-595b-4a5e-b9f6-30137f86d5d9

and the full release video: https://www.youtube.com/watch?v=bmJSLPYrKtE

We're excited to hear feedback and see the community build on top of it!

Quick links:
**Discord:** https://discord.gg/ZzsKTAKM
**Github:** https://www.github.com/camb-ai/mars5-tts
**Website:** https://www.camb.ai/
**Youtube:** https://www.youtube.com/@camb-ai

**Comment by nateshmbhat** [comment]
Does this support training a single voice with hours if audio content and generating any length of text?
